core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -110,3 +114,60 @@ describe('Customized profile page for current user', function() {
     general.checkForConsoleErrors([]);
   });
 });
+
+describe('View explorations created by user in their profile page',
+  function() {
+    var TEST_USERNAME = 'myUser';
+    var TEST_EMAIL = TEST_USERNAME + '@example.com';
+
+    var ANOTHER_USERNAME = 'anotherUser';
+    var ANOTHER_EMAIL = ANOTHER_USERNAME + '@example.com';
+
+    var profilePage = null;
+    var creatorDashboardPage = null;
+
+    var EXPLORATION = {
+      title: 'A new exploration',
+      category: 'Learning',
+      objective: 'The goal is to create a new exploration',
+      language: 'English'
+    };
+
+    beforeAll(function() {
+      profilePage = new ProfilePage.ProfilePage();
+      creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    });
+
+    it('another user should create a new exploration', function() {
+      users.createUser(ANOTHER_EMAIL, ANOTHER_USERNAME);
+      users.login(ANOTHER_EMAIL);
+
+      workflow.createAndPublishExploration(
+        EXPLORATION.title,
+        EXPLORATION.category,
+        EXPLORATION.objective,
+        EXPLORATION.language
+      );
+
+      creatorDashboardPage.get();
+      creatorDashboardPage.expectToHaveExplorationCard(EXPLORATION.title);
+    });
+
+    it('user should view the exploration in another user\'s profile',
+      function() {
+        users.createUser(TEST_EMAIL, TEST_USERNAME);
+        users.login(TEST_EMAIL);
+
+        profilePage.get(ANOTHER_USERNAME);
+        profilePage.expectToHaveExplorationCards();
+        profilePage.expectToHaveExplorationCardByName(EXPLORATION.title);
+      });
+
+    afterEach(function() {
+      users.logout();
+      general.checkForConsoleErrors([
+        'Failed to load resource: the server responded with a status of 404'
```

Comment:
What if there are other errors?

Reply:
How can I discover what errors can be thrown? The error in array was the unique that showed up on terminal when running the tests.

Reply:
@nithusha21 can you advise?

Reply:
@marianazangrossi, Can you explain why do we expect to have this console error in the first place?

Reply:
@DubeySandeep I checked out why this error is happening.

When the first ` it block` runs, this error is thrown because `third_party/static/skulpt-0.10.0/skulpt-stdlib.js` doesn't exist on my computer. I tried to install it running `install_third_party_libs.py` but just `skulpt.min.js` is install on `third_party/static/skulpt-0.10.0` path.
Just to be sure, I commented the script tag in `skulpt.html` that was importing the missing script. Then I removed the error from the array param. The test ran sucessfully.

I don't have any experience with Python at all, so I couldn't understand what part is not right. Do you have any idea? Does someone have faced this problem before?

Reply:
@marianazangrossi Thanks for checking, I see that's because of some missing dependencies, you can remove the element from `checkForConsoleErrors`. 

@kevinlee12, Can you please check why the `skulpt-stdlib.js` is not getting downloaded? (I can see the same file in my local repo.)

Reply:
It could have been a network error. since it looks like this error isn't there any more.

Reply:
Done